README.md

```diff
@@ -19,7 +19,42 @@ using AbstractDifferentiation
 
 ## `AbstractDifferentiation` backends
 
-To use `AbstractDifferentiation`, first construct a backend instance `ab::AD.AbstractBackend` using your favorite differentiation package in Julia that supports `AbstractDifferentiation`. For higher order derivatives, you can build higher order backends using `AD.HigherOrderBackend`. For instance, let `ab_f` be a forward-mode automatic differentiation backend and let `ab_r` be a reverse-mode automatic differentiation backend. To construct a higher order backend for doing forward-over-reverse-mode automatic differentiation, use `AD.HigherOrderBackend((ab_f, ab_r))`. To construct a higher order backend for doing reverse-over-forward-mode automatic differentiation, use `AD.HigherOrderBackend((ab_r, ab_f))`.
+To use `AbstractDifferentiation`, first construct a backend instance `ab::AD.AbstractBackend` using your favorite differentiation package in Julia that supports `AbstractDifferentiation`.
+The following backends are made available by `AbstractDifferentiation` if their corresponding package is loaded (thanks to [Requires.jl](https://github.com/JuliaPackaging/Requires.jl)):
+
+- `AD.ForwardDiffBackend()` for [ForwardDiff.jl](https://github.com/JuliaDiff/ForwardDiff.jl)
+- `AD.FiniteDifferencesBackend()` for [FiniteDifferences.jl](https://github.com/JuliaDiff/FiniteDifferences.jl)
+- `AD.ReverseDiffBackend()` for [ReverseDiff.jl](https://github.com/JuliaDiff/ReverseDiff.jl)
+- `AD.ReverseRuleConfigBackend(ruleconfig)` for any [ChainRules.jl](https://github.com/JuliaDiff/ChainRules.jl)-compatible reverse mode backend
+- `AD.ZygoteBackend()` for [Zygote.jl](https://github.com/FluxML/Zygote.jl)
+- `AD.TrackerBackend()` for [Tracker.jl](https://github.com/FluxML/Tracker.jl)
+
+Here's an example:
+
+```julia
+julia> using AbstractDifferentiation
+
+julia> ab = AD.ZygoteBackend()
+ERROR: UndefVarError: ZygoteBackend not defined
+Stacktrace:
+ [1] getproperty(x::Module, f::Symbol)
+   @ Base ./Base.jl:35
+ [2] top-level scope
+   @ REPL[3]:1
+
```

Comment:
I'm not sure if this is helpful. Maybe sufficient to just show how it works?
```suggestion
```

Reply:
I think it helps to underline the fact that AbstractDifferentiation won't load your package for you. Again, speaking from a beginner's perspective, this is something that I would love to know right away, and this example shows it clearly.

Reply:
Sure, but isn't documenting this and showing the correct code sufficient? It still seems strange to me to include a failing example here. I think I don't know any package with such a Requires-setup that does show how to not do it.

Reply:
Your call. I just thought that it may help users unfamiliar with Requires, but maybe it does more harm than good indeed